cobra/core/metabolite.py

```diff
@@ -237,9 +237,11 @@ def summary(self, solution=None, threshold=0.01, fva=None, names=False,
             Format string for floats (default '.3g').
 
         """
-        from cobra.flux_analysis.summary import metabolite_summary
-        return metabolite_summary(self, solution=solution, threshold=threshold,
-                                  fva=fva, names=names, floatfmt=floatfmt)
+
+        from cobra.core.summary import MetaboliteSummary
```

Comment:
Are you getting cyclic dependencies or why do you import inside the method?

Reply:
I was getting cyclic import when it was in `flux_analysis`. I'll check if it is still facing the same issue in `core`, if not I'll place the import at the top.

Reply:
The issue persists, so keeping it as it is.

Reply:
`MetaboliteSummary` and `ModelSummary` require some imports from `cobra.core` so that is again in turn becoming cyclic.